js/hoot/control/utilities/ImportDataset.js

```diff
@@ -454,7 +454,7 @@ Hoot.control.utilities.importdataset = function(context) {
         } else if(typeName === 'OSM') {
             d3.select('#ingestfileuploader')
             .property('multiple', 'false')
-            .attr('accept', '.osm')
+            .attr('accept', '.osm,.osm.zip')
```

Comment:
I tested in both Firefox and Chrome (recent versions) and they will both default the file chooser to the accept types you specify.  You can still upload a .zip osm file by choosing the custom files option in the file chooser.  The file inside needs to have a .osm file extension.  I more or less did that to drive home the point this option is for zipped osm files.  Maybe that's not necessary?